Makefile

```diff
@@ -575,16 +575,19 @@ test-benchmark: ## Run the benchmark tests -- Note that this can only be ran for
 	@echo "The pprof files generated are: cpu.prof and mem.prof"
 
 .PHONY: e2e
-e2e: e2e-set-image prep-e2e ## Run full end-to-end tests that exercise content on an operational cluster.
-	@CONTENT_IMAGE=$(E2E_CONTENT_IMAGE_PATH) BROKEN_CONTENT_IMAGE=$(E2E_BROKEN_CONTENT_IMAGE_PATH) $(GO) test ./tests/e2e $(E2E_GO_TEST_FLAGS) -args $(E2E_ARGS) | tee tests/e2e-test.log
+e2e: e2e-set-image prep-e2e e2e-parallel e2e-test-wait e2e-serial ## Run full end-to-end tests that exercise content on an operational cluster.
```

Comment:
This target still isn't used by CI since it invokes the parallel and serial tests separately, but it's around in case people are using it locally for a complete e2e run.